etc/docker/kernel-spark-py/Dockerfile

```diff
@@ -1,7 +1,10 @@
 FROM elyra/spark-py:v2.4.0
 
-RUN apk add --no-cache build-base libffi-dev openssl-dev python-dev && \
-    pip install cffi ipykernel ipython jupyter_client pycrypto
+RUN apk del python3 && \
```

Comment:
Did we need to remove python3? The next line we install python3-dev in which python3 is a dependency, so I think we would end up re-downloading it anyway

Reply:
thanks for the explanation, LGTM